pkg/pb/service.proto

```diff
@@ -98,4 +102,22 @@ service MyApp {
         get: "/requests/{service}"
     };
   }
+
+  rpc GetMode (GetModeRequest) returns (GetModeResponse){
+    option (google.api.http) = {
+        get: "/mode"
+    };
+  }
+
+  rpc SetMode (SetModeRequest) returns (SetModeResponse){
+    option (google.api.http) = {
+        get: "/mode/{mode}"
```

Comment:
Get action should not change data. There is a lot of [useful info](https://softwareengineering.stackexchange.com/questions/188860/why-shouldnt-a-get-request-change-data-on-the-server) about that case.